test/operators.js

```diff
@@ -827,12 +826,11 @@ prepareAndRunTest('support live offset operations', dir, (t, db, raf) => {
       })
     ),
     toPullStream(),
-    pull.filter((x) => x), // filter out the first non-live empty result
     pull.drain((msg) => {
-      t.equal(msg.value.author, bob.id)
+      t.equal(msg.value.author, alice.id)
 
       // test we don't get live messages after aborting stream
-      addMsg(state.queue[1].value, raf, (e2, msg2) => {
+      addMsg(state.queue[0].value, raf, (e2, msg2) => {
```

Comment:
What's going on here, why are we adding `state.queue[0].value` twice, and where did bob go?

Reply:
It should do the same thing as before, we just don't need to skip the values. Before it was also adding the same message 2 times. I'll add back bob because that should make the test easier to read.

Reply:
Oh ok I see. Thanks